app/src/components/common/Button.vue

```diff
@@ -65,4 +65,17 @@ button {
   border-color: #327136;
   color: #327136;
 }
+
+.disabled {
+  background: #ccc;
+  border-color: #ccc;
+  color: #666;
+  cursor: not-allowed;
+}
+
+.disabled:hover {
+  background: #ccc;
+  border-color: #ccc;
+  color: #666;
+}
```

Comment:
Maybe you find it useful.

```suggestion
.disabled,
.disabled:hover {
  background: #ccc;
  border-color: #ccc;
  color: #666;
}

/* Specific property for the normal (non-hover) state */
.disabled {
  cursor: not-allowed;
}
```

Reply:
I dont think the style of disabled button on hover should be different